modules/affinity/src/main/java/org/apache/ignite/internal/affinity/AffinityManager.java

```diff
@@ -88,7 +88,9 @@ public AffinityManager(
         metaStorageMgr.registerWatchByPrefix(new ByteArray(INTERNAL_PREFIX), new WatchListener() {
             @Override public boolean onUpdate(@NotNull WatchEvent watchEvt) {
                 for (EntryEvent evt : watchEvt.entryEvents()) {
-                    String tabIdVal = evt.newEntry().key().toString().substring(INTERNAL_PREFIX.length());
+                    String keyAsString = new String(evt.newEntry().key().bytes(), StandardCharsets.UTF_8);
```

Comment:
I'd rather use 
```
new ByteArray(evt.newEntry().key().bytes()).toString()
```
in order to hide StandardCharsets.UTF_8

Reply:
Done.